conf/eddie.config

```diff
@@ -0,0 +1,50 @@
+//Profile config names for nf-core/configs
+params {
+  config_profile_description = 'University of Edinburgh (eddie) cluster profile provided by nf-core/configs.'
+  config_profile_contact = 'Alison Meynert (@ameynert)'
+  config_profile_url = 'https://www.ed.ac.uk/information-services/research-support/research-computing/ecdf/high-performance-computing'
+}
+
+executor {
+  name = "sge"
+  queueSize = "100"
+}
+
+process {
+  clusterOptions = { task.memory ? "-l h_vmem=${task.memory.bytes/task.cpus}" : null }
+  scratch = true
+  penv = { task.cpus > 1 ? "sharedmem" : null }
+
+  // common SGE error statuses
+  errorStrategy = {task.exitStatus in [143,137,104,134,139,140] ? 'retry' : 'finish'}
+  maxErrors = '-1'
+  maxRetries = 3
+
+  beforeScript =
+  """
+  . /etc/profile.d/modules.sh
+  module load 'roslin/singularity/3.5.3'
+  export SINGULARITY_TMPDIR="\$TMPDIR"
```

Comment:
This _may_ cause some issues if people are using this env variable themselves, or the `NXF_SINGULARITY_CACHEDIR` variable that we now recommend: https://nf-co.re/usage/introduction#tips-and-tricks

We use this more and more with DSL2 to have a shared directory so that multiple pipelines can use the same DSL2 containers.

Reply:
We recommend in our documentation to use the `NXF_SINGULARITY_CACHEDIR` variable and have some shared folders set up to take advantage. Setting `SINGULARITY_TMPDIR` explicitly is because of a poorly configured set up on our cluster for temporary directories whereby some nodes end up writing everything to /tmp and overfilling it. We know that `$TMPDIR` is always correctly set. This can be updated in future if the configuration gets fixed.

Reply:
ok great - and this doesn't take priority over `NXF_SINGULARITY_CACHEDIR` if set then? Good to know!